spec/controllers/ems_common_controller_spec.rb

```diff
@@ -232,6 +232,57 @@ def test_setting_few_fields
         post :button, :params => { :pressed => "vm_edit", :format => :js, "check_#{vm.id}" => "1" }
         expect(controller.send(:flash_errors?)).not_to be_truthy
       end
+
+      context 'displaying nested lists from summary page of container provider' do
+        let(:provider) { ManageIQ::Providers::ContainerManager.new }
+
+        before do
+          allow(controller).to receive(:javascript_redirect).and_return(true)
+          allow(controller).to receive(:performed?).and_return(true)
+          controller.instance_variable_set(:@display, display)
+          controller.instance_variable_set(:@_params, :pressed => press, :miq_grid_checks => item.id.to_s, :id => provider.id)
+          controller.instance_variable_set(:@breadcrumbs, [])
+        end
+
+        {
+          'container_image'      => 'Container Images',
+          'container_replicator' => 'Container Replicators',
+          'container_node'       => 'Container Nodes',
+          'container_group'      => 'Container Pods'
+        }.each do |display_s, items|
```

Comment:
You might not need the hash values, just use an array and then call `humanize.pluralize` on each item.

Reply:
But.. what about Container Pods?

Reply:
Do you really need it in the testing, or just in the describe/context/it strings?

Reply:
If I removed the hash values and replaced them by an array of four items, I don't know how to set the string for context block properly and nicely (to see there for example 'tagging selected Container Pods' and not 'tagging selected Container Groups'). It's only for this there - for info what is being tested. 

I consider this solution as the easiest and the most readable (at least for now). If something went wrong in some of the specs, we can easily recognize, what went wrong: we will not need to figure out that `container_group` is related to _Container Pods_ because it's already there. And I need all of the contexts there to set `params[:pressed]` properly for each situation.

Reply:
ok